<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtNewSyncDialog</class>
 <widget class="QDialog" name="QtNewSyncDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>88</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Sync</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtClient.qrc">
    <normaloff>:/Resources/icon.png</normaloff>:/Resources/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="encryptedBox">
     <property name="text">
      <string>encrypted</string>
     </property>
     <property name="icon">
      <iconset resource="qtClient.qrc">
       <normaloff>:/Resources/lock.png</normaloff>:/Resources/lock.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QGridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sendLabel">
       <property name="text">
        <string>&lt;i&gt;sending request...&lt;/i&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>To create a new Sync choose a Name and whether to encrypt files</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qtClient.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>QtNewSyncDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QtNewSyncDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
